Live page docs update (#632)
pydantic/logfire: docs/guides/web-ui/live.md

# Live View

> The live view is a central part of **Logfire**, where you can see traces in real-time.

Reviewer: ```suggestion
The live view is the focal point of **Logfire**, where you can see traces arrive in real-time.
```

The live view is useful for watching what's going on within your

> application in real-time (as the name suggests), but it can also be used to explore historical data via the *live view search*.

Reviewer: ```suggestion
application in real-time (as the name suggests). You can also explore historical data in the **search pane**.
```

> ## The Live View Search

Reviewer: ```suggestion
## SQL search pane
```

> To view the search box, click on `Search your spans` (keyboard shortcut `/`), this

Reviewer: ```suggestion
To search the live view, click `Search your spans` (keyboard shortcut `/`), this
```

> will open the search box:

Reviewer: ```suggestion 
opens the search pane:
```

![Search box](../../images/guide/live-view-search.png)

### SQL Search

> Here you can write raw SQL. A good choice if you know what you want and/or love SQL. As the greyed out
> `SELECT * FROM RECORDS WHERE` implies, this is the `WHERE` clause of a SQL query.
> It has auto-complete & schema hints, so try typing something to get a reminder. To run your query click `Run` or keyboard
> shortcut `cmd+enter` (or `ctrl+enter` on windows/linux).

Reviewer: ```suggestion
For confident SQL users, write your queries directly here. For devs who want a bit of help, try the new [PydanticAI](https://ai.pydantic.dev/) feature which generates a SQL query based on your prompt. You can also review the fields available and populate your SQL automatically using the `Reference` list, see more on this below.

**
As the greyed out `SELECT * FROM RECORDS WHERE` implies, you're searching inside the `WHERE` clause of a SQL query. It has auto-complete & schema hints, so try typing something to get a reminder. To run your query click `Run` or keyboard shortcut `cmd+enter` (or `ctrl+enter` on windows/linux).
```

new query with `Get new SQL` or (if you have some SQL already) modify a work-in-progress query with `Modify existing SQL`.
You'll note that whenever you click either of these buttons the SQL is updated in the SQL Search box above.

3. **Feedback button:** Click the feedback button to provide us feedback.
[TODO: Add image of natural language query]

4. **Light/Dark mode toggle:** Cycles between light, dark, and system — because everyone seems to have an opinion on this :smile:
Under the hood this uses
an LLM running with [PydanticAI](https://github.com/pydantic/pydantic-ai).

5. **Link to the current view:** Clicking this copies a link to the page you are on, with the same query etc.
### Common Clauses
Common clauses: A list of pre-populated useful queries. A powerful way to rapidly generate the query you need, whilst
simultaneously learning more about all the ways you can search your data. Clicking multiple clauses will add them to
your query with an `AND` statement. If you'd like something other than an `AND` statement you can either modify the
SQL populated in the SQL search or use the [explore screen](explore.md) for more complex queries.

> 6. **Organization selection panel:** Opens a drawer with links to the different organizations you are a member of, and also has links to the Terms and Conditions, Support, Documentation, and a Log Out button.

Author: I've deliberately culled a lot of these - some are no longer relevant, but the section is currently just a bit too much of a "wall of text". I've tried to pull out the key/least intuitive areas to describe
